apothecary/formulas/FreeImage/FreeImage.sh

```diff
@@ -61,13 +62,14 @@ function prepare() {
 	    cat > Source/LibRawLite/src/swab.h << ENDDELIM
 	    #include <stdint.h>
         #include <asm/byteorder.h>
-        inline void swab(const void *from, void*to, ssize_t n)
+		#define ___swab(x)  ({ __u16 __x = (x);   ((__u16)(   (((__u16)(__x) & (__u16)0x00ffU) << 8) | (((__u16)(__x) & (__u16)0xff00U) >> 8) ));  })
```

Comment:
__arch_swab16  no longer works, i found a different repo that used this as a workaround, but not sure if it's correct.